### PR TITLE
LPS-174056 [Raylife AP] Enable Account Functionality for User Avatar

### PR DESCRIPTION
**Pull request by raquelpaiva**
Old PR [LPS-174056](https://github.com/pt-liferay-solutions/liferay-portal/pull/3562)

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/layout/index.html [hunk 1]
```diff
@@ -1,61 +1,110 @@
 <div class="bg-brand-primary-lighten-6 d-flex layout-container">
-<div class="bg-brand-primary-darken-5 d-lg-flex d-none flex-column justify-content-between ray-sidebar-container ray-sticky-top sticky-top">
+	<div
+		class="bg-brand-primary-darken-5 d-lg-flex d-none flex-column justify-content-between ray-sidebar-container ray-sticky-top sticky-top"
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/layout/index.html:3]: Do you really need "ray-sticky-top sticky-top"?

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/avatar-dropdown-menu/index.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
 			/]
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/avatar-dropdown-menu/index.html:10]: Please rename this commit to _LPS-176972 Adjustment position dropdown_

**Review by joanagrego — CHANGES_REQUESTED** [review]

**Comment by joanagrego** [comment]
ci:test:sf

**Review by joanagrego — CHANGES_REQUESTED** [review]
All your commits should have the LPS pattern. Please add it to your last commit.

**Comment by joanagrego** [comment]
Just starting reviewing :)

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/layout/index.html (hunk 1 above)
Comment (raquelpaiva) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/layout/index.html]: Yes.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/avatar-dropdown-menu/index.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	position: absolute;
+}
+
+.dropdown .dropdown-content span {
+	color: #6b6c7e;
+	font-weight: bold;
+}
+
+.dropdown .dropdown-content a {
+	color: #6b6c7e;
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/avatar-dropdown-menu/index.css:18]: Please, use dialect for this.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/avatar-dropdown-menu/index.html (hunk 2 above)
Comment (raquelpaiva) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/avatar-dropdown-menu/index.html:10]: Done!

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/layout/index.css [hunk 4]
```diff
@@ -36,6 +36,7 @@
 .ray-sidebar-container .footer-bar .avatar-container,
 .ray-sidebar-container .middle-bar .menu-item-name {
 	overflow: hidden;
+	padding-left: 5px;
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/layout/index.css:39]: Please, use bootstrap for this `padding-left`.
```suggestion
	
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/layout/index.css [hunk 5]
```diff
@@ -167,7 +170,7 @@
 	font-weight: 700;
 }
 
-@media screen and (max-width: 1625px) {
+@media creen and (max-width: 1625px) {
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/layout/index.css:173]: Please, believe this change was made in the wrong way. 
```suggestion
@media screen and (max-width: 1625px) {
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/layouts/1_dashboard/page-definition.json [hunk 6]
```diff
@@ -61,21 +64,42 @@
...
+								"definition": {
+								},
+								"pageElements": [
+									{
+										"definition": {
+											"widgetInstance": {
+												"widgetConfig": {
+													"properties": "route=notification-sidebar"
+												},
+												"widgetName": "com_liferay_client_extension_web_internal_portlet_ClientExtensionEntryPortlet_20096_RAYLIFEAGENTPORTAL001"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/layouts/1_dashboard/page-definition.json:95]: Please, replace all the widget names in all `page-definitions.json`.

```suggestion
												"widgetName": "[$CLIENT_EXTENSION_ENTRY_ID:RAYLIFEAGENTPORTAL$]"
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/layouts/10_notifications-list/page-definition.json [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+						"paddingTop": 0,
+						"widthType": "Fluid"
+					}
+				},
+				"pageElements": [
+					{
+						"definition": {
+							"fragment": {
+								"key": "layout",
+								"siteKey": "AP1"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/layouts/10_notifications-list/page-definition.json:26]: Please, change all the siteKey properties in all page-definitions of the Raylife AP.

```suggestion
								"siteKey": "[$GROUP_KEY$]"
```

**Comment by joanagrego** [comment]
Resent --> [PR](https://github.com/pt-liferay-solutions/liferay-portal/pull/3587)

**Pull request closed without merging**